### PR TITLE
Prototype react app

### PR DESCRIPTION
**Pull request by alexandru-damian**

### CODE REVIEW

File: app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+const baseURL = "https://nab-grinder-converter.herokuapp.com";
+export default class App extends Component {
+  state = {
+    data: [],
+  };
+
+  componentDidMount() {
```
Comment (Ose-Me) [inline app/src/App.js:11]: Can this be done inside the Converter JSX component

File: app/src/components/Router.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import Navbar from "./Navbar";
+import App from "../App"
+import About from "../pages/About"
+import FAQ from "../pages/FAQ";
+
+export default function Router ()
+{
+    return(
+    <BrowserRouter>
+    <Routes>
```
Comment (Ose-Me) [inline app/src/components/Router.js:9]: The layout of the main page has all the content in one page. This type of routing might be best suited when we add the admin to the project.

File: app/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# Converter app
```
Comment (Ose-Me) [inline app/README.md]: There are two readme files. You can update this one instead and delete the duplicate files in the Index folder. Then just move the contents in the app folder to the main folder.

File: app/src/App.js (hunk 1 above)
Comment (alexandru-damian) [reply app/src/App.js:11]: It can, but its still a prototype so the focus is functionality

File: app/src/components/Router.js (hunk 2 above)
Comment (alexandru-damian) [reply app/src/components/Router.js:9]: You mean to use anchors as in the previous version?

File: app/README.md (hunk 3 above)
Comment (alexandru-damian) [reply app/README.md]: Yeah, that could work.

File: app/src/components/Router.js (hunk 2 above)
Comment (Ose-Me) [reply app/src/components/Router.js:9]: Yes anchor tags since all the content is in one page

File: app/src/App.js (hunk 1 above)
Comment (Ose-Me) [reply app/src/App.js:11]: I asked because of easier state management
But since this is just for testing the functionality then it's fine

**Comment by alexandru-damian** [comment]
@Ose-Me  The App.css can be ignored it was to quickly test the anchors

**Review by Ose-Me — APPROVED** [review]
The folder structure would still need to be adjusted in the near future

**Pull request merged**